pages/settings.js

```diff
@@ -236,6 +237,26 @@ export default function Settings ({ ssrData }) {
             name='hideInvoiceDesc'
             groupClassName='mb-0'
           />
+          <Checkbox
+            label={
+              <div className='d-flex align-items-center'>auto forget withdrawal invoices after 7 days
+                <Info>
+                  <ul className='fw-bold'>
+                    <li>use this to protect receiver privacy</li>
+                    <li>applies retroactively, cannot be reversed</li>
+                    <li>withdrawal invoices are kept at least 7 days:
+                      <ul>
+                        <li>for debugging purposes</li>
+                        <li>to prevent a thief from covering his tracks</li>
```

Comment:
i would just mention: `withdrawal invoices are kept at least 7 days for security and debugging purposes`

this also doesn't directly refer to the user to be a potential thief, lol